napari/plugins/pypi.py

```diff
@@ -114,26 +107,27 @@ def ensure_published_at_pypi(
         if any(f'Development Status :: {1}' in x for x in classifiers):
             return None
 
-    return ProjectInfo(
+    return PackageMetadata(
+        metadata_version="1.0",
         name=normalized_name(info["name"]),
         version=info["version"],
-        url=info["home_page"],
         summary=info["summary"],
+        home_page=info["home_page"],
         author=info["author"],
         license=info["license"] or "UNKNOWN",
     )
 
 
 def iter_napari_plugin_info(
     skip={'napari-plugin-engine'},
-) -> Generator[ProjectInfo, None, None]:
+) -> Generator[PackageMetadata, None, None]:
```

Comment:
Type hint needs to be updated to match Tuple return type

Reply:
Fixed